doc/conf.py

```diff
@@ -63,7 +63,10 @@
 #source_encoding = 'utf-8-sig'
 
 # The master toctree document.
-master_doc = 'index'
+if tags.has("kerneldoc"):
+    master_doc = 'kernel'
```

Comment:
This won't result in an ``index.html`` for the doc root in the generated website.  I'm not sure where you intend to publish this html tree, or is the intention just to create a PDF file for "Kernel Only" documentation?

Reply:
This will probably be an easy fix to generate the index.html file.